cli/dcoscli/node/main.py

```diff
@@ -3,19 +3,24 @@
 Usage:
     dcos node --info
     dcos node [--json]
+    dcos node log [--follow --lines=N --master --slave=<slave-id>]
```

Comment:
Should this be `[--follow --lines=N] --master | --slave=<slave-id>`? Not sure how I feel about "required optional" flags. Do we have an example of a CLI that does this? cc @ConnorDoyle @ssk2 @tamarrow .